core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -145,14 +154,14 @@ class AdminClient(val time: Time,
 
   def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
     val group = describeGroup(groupId)
```

Comment:
@vahidhashemian did you mean when describeGroup returns null? Because that case has been handled...